### ISSUE TITLE
Footer saknas

### ISSUE THREAD

**Issue by SA6JKK** [ETAChalmers/ETA-Hemsida-3.0#8]
Footer saknas. Footer borde finnas

**Comment by almbr** [comment]
Jo. Har väntat med det för att jag inte varit säker på vad man ska ha där. Vad tycker du @KonstigSak?

Alternativ jag tänkt:
- Länkar (navbar) till andra delar av hemsidan
- En "Del av Chalmers Studentkår" logga grej
- Länkar till systerföreningar. Typ CRF, XP, Case.
- Kontaktuppgifter. Som på existerande hemsida + organisationsnummer.
- Sponsorer?

Vad tycks?

**Comment by SA6JKK** [comment]
Yes, låter ok

- Navbar/länkar är viktigt
- Del av Chalmers Studentkår är viktig
- Länk till systerföreninger gillar jag inte
- Kontaktuppgifter är mestast viktigast
- sponsorer kan vi ha "över" allt annat. 

ish såhär

…......... | SPONSORER | …............
länkar | Chalmers logga | kontakt

**Comment by almbr** [comment]
Gjorde en första prototyp på footer. Mest för att leka lite med SCSS. Ganska fult hittills tycker jag.

Chalmers stundentkår-loggan är vald utefter deras [styleguide](https://chalmersstudentkar.se/wp-content/uploads/2017/09/Chalmers-Studenk%C3%A5rs-Grafiska-Profil_2.pdf). Kanske ska köra på någon annan dock.

Har inte pillat något med sponsor-idéer som ni ser.

![image](https://user-images.githubusercontent.com/20804017/54731095-a0fdc100-4b8c-11e9-9f29-fd07f6f947ba.png)

**Comment by almbr** [comment]
Feedback:
- Gör den svart
- Gör den typ som http://etek.chalmers.se/
- Använd annan ChS logo
